homeassistant/components/zha/core/device.py

```diff
@@ -592,12 +592,13 @@ async def async_initialize(self, from_cache: bool = False) -> None:
         self.debug("started initialization")
         await self._zdo_handler.async_initialize(from_cache)
         self._zdo_handler.debug("'async_initialize' stage succeeded")
-        await asyncio.gather(
-            *(
-                endpoint.async_initialize(from_cache)
-                for endpoint in self._endpoints.values()
-            )
-        )
+
```

Comment:
should we add a comment to remind ourselves (and others) that this is intentionally not a gather?